WebViewControl/WebView.JavascriptExecutor.cs

```diff
@@ -163,7 +163,7 @@ internal class JavascriptExecutor : IDisposable {
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"{nameof(EvaluateScript)} '{script}' on ('{Id}')");
 #endif
-                var evaluationTask = new TaskCompletionSource<T>(TaskContinuationOptions.RunContinuationsAsynchronously);
+                var evaluationTask = new TaskCompletionSource<T>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
can you explain this change?

Reply:
We were using the wrong enumeration. (It happens that they both have the same value: 64)

Please check: https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskcompletionsource-1.-ctor?view=net-7.0#system-threading-tasks-taskcompletionsource-1-ctor(system-threading-tasks-taskcreationoptions)